Makefile

```diff
@@ -50,11 +57,63 @@ fields:
 	cat fields.tmp.yml >> fields.yml
 	rm -f fields.tmp.yml fields.tmp.yml.bak
 
-docs:
-	if [ ! -d $(PWD)/build/docs ]; then \
-		git clone --depth=1 https://github.com/elastic/docs.git ./build/docs ; \
-	fi
+# Format code and files in the repo.
+.PHONY: fmt
+fmt: ve
+	$(FIND) -name '*.py' -exec build/ve/bin/autopep8 --in-place --max-line-length 120 {} \;
+	go get golang.org/x/tools/cmd/goimports
+	goimports -w -l -local github.com/elastic $(shell $(FIND) -name '*.go')
 
-	./build/docs/build_docs.pl --doc ./docs/index.asciidoc --chunk=1 $(OPEN_DOCS) -out ./build/html_docs
+# Alias to generate everything.
+.PHONY: generate
+generate: csv readme template fields
+
+# Check Makefile format.
+.PHONY: makelint
+makelint: SHELL:=/bin/bash
+makelint:
+	@diff <(grep ^.PHONY Makefile | sort) <(grep ^.PHONY Makefile) \
+	  || echo Makefile targets need to be sorted.
```

Comment:
Someone doesn't like unsorted Makefiles? :-) Only downside to this is that we can't group together commands that are similar.